zk/hermez_db/db.go

```diff
@@ -1887,3 +1889,18 @@ func (db *HermezDbReader) getForkIntervals(forkIdFilter *uint64) ([]types.ForkIn
 
 	return forkIntervals, nil
 }
+
+func (db *HermezDb) WriteBadTxHashCounter(txHash common.Hash, counter uint64) error {
```

Comment:
Do we ever delete them?

Reply:
We could add in a cleanup routine somehow to do this for sure, but in another PR perhaps after this one.